schemas/data/launches.example.1.json

```diff
@@ -0,0 +1,6 @@
+{
+  "schema:name": "application metric: launches",
+  "@id": "https://ns.adobe.com/xdm/metrics/launches",
+  "xdm:measurement": "count",
```

Comment:
does this indicate a rolling count of launches will be kept with profile? I was under the impression that launches was only a hit that could go in and just sit in there saying a launch happened.

Reply:
It is only a hit. Services on top will know to treat it like a count.